tests/integration/targets/ansible_galaxy_install/aliases

```diff
@@ -1,3 +1,4 @@
 destructive
 shippable/posix/group3
 skip/python2.6
+context/controller  # While this is not really true, this module mainly is run on the controller, *and* needs access to the ansible-galaxy CLI tool
```

Comment:
Although this test is for a module, the module can only run on the controller, so I'd rephrase this comment:

```suggestion
context/controller  # this module only runs on the controller since it needs access to the ansible-galaxy CLI tool
```

Reply:
It can also run on the target - assuming ansible-core/ansible-base/Ansible 2.9 has been installed there.